.github/workflows/e2e_tests.yaml

```diff
@@ -371,6 +371,6 @@ jobs:
         run: melos bootstrap --scope tests && melos bootstrap --scope "cloud_firestore*"
       - name: "Install Tools"
         run: |
-          npm install -g firebase-tools
+          npm install -g firebase-tools 
```

Comment:
Intentional change? 👀

Reply:
Oups, was trying to fix an issue with the emulator, that isn't happening anymore, I've removed :)